server/zclient.go

```diff
@@ -654,41 +669,27 @@ func (z *zebraClient) loop() {
 						"Topic": "Zebra",
 					}).Debug("MultiplePathListLen %d, PathListLen %d", len(msg.MultiPathList), len(msg.PathList))
 					for _, dst := range msg.MultiPathList {
-						if len(dst) == 0 {
-							continue
-						}
-						path := dst[0]
+						// We need to check all of Path for self route withdraw.
```

Comment:
Is the logic changed in this PR? If so then I would suggest adding some tests to verify it.

Reply:
test has been addded.